File: ojuz/ioi/2014/Day2/holiday-templates/grader.pas
program main;

uses holiday;

var 
    attraction : array[0..99999] of longint;
    n, start, d, i: longint;

begin
    read(n, start, d);
    for i:=0 to n-1 do
    begin
	read(attraction[i]);
    end;
    writeln(findMaxAttraction(n, start, d, attraction));
end.

File: ojuz/ioi/2014/Day2/holiday-templates/grader.c
#include<stdio.h>
#include"holiday.h"

int main() {
    int n, start, d;
    int attraction[100000];
    int i, n_s;
    n_s = scanf("%d %d %d", &n, &start, &d);
    for (i = 0 ; i < n; ++i) {
	n_s = scanf("%d", &attraction[i]);
    }
    printf("%lld\n", findMaxAttraction(n, start, d, attraction));
    return 0;
}
